pkg/service/tools/service.go

```diff
@@ -1,12 +1,90 @@
 // Package tools provides utility functions for the service package.
 package tools
 
-import "context"
+import (
+	"context"
+	"fmt"
+	"os"
+)
 
 type Service interface {
 	Update(ctx context.Context) error
 	CreateConfig(ctx context.Context, filePath string, config string) error
+	Normalise(ctx context.Context, path string, testSet string, testCases string) error
 }
 
 type teleDB interface {
 }
+type TestCaseFile struct {
```

Comment:
I think this is already present in models

Reply:
In models we have a strut for TestCase, not one for the entire testcase yaml file. Having this struct makes it easier for unmarshalling the testcase file to a struct and back to the yaml file.